docs/index.md

```diff
@@ -0,0 +1,51 @@
+<h1>ddev Documentation</h1>
+
+[ddev](https://github.com/drud/ddev) is a local web development environment management system powered by Docker and Docker Compose. It provides rapid, repeatable, and destructable environments geared for Drupal and WordPress development.
+
+## System Requirements
+
+- [docker](https://www.docker.com/community-edition)
+- OS Support
+  - macOS Sierra (fully supported with automated tests)
+  - Linux
+    * Ubuntu 16.04 LTS (fully supported with automated tests)
+    * Debian Jessie (tested manually with automated tests planned)
+    * Fedora 25 (tested manually with automated tests planned)
+  - Windows 10 Pro (**experimental support!**)
+    * See [Decisions and Plan for Linux & Windows Support](https://github.com/drud/ddev/issues/196#issuecomment-300178008) for more information and the various options on getting ddev operational on Windows 10.
+
+We are open to expanding this list to include additional OSs as well as improve our existing support for the ones listed above. Please [let us know](https://github.com/drud/ddev/issues/new) if you hit an issue!
+
+### Using ddev with other development environments
+ddev requires ports 80 and 3306 to be available for use on your system when sites are running. If you are using another local development environment along side ddev, please ensure the other environment is turned off or otherwise not occupying ports 80 and 3306.
```

Comment:
It's stated in the readme and this docs home page, since system requirements and install are part of both. I think it makes sense to have in both places since either could be the introductory point for a user.